styleguide/components/31-molecules/fields/checkbox-with-filter/_checkbox-with-filter.scss

```diff
@@ -62,23 +82,9 @@
       @include theme('border-color', 'color-primary--lighten-4', 'checkbox-with-filter-action-border-bottom');
 
       display: flex;
-      position: fixed;
-      bottom: 0;
-      left: 0;
       align-items: center;
-      width: 100vw;
-      height: 3rem;
-      margin-top: -3rem;
-      padding-left: 1rem;
       border-top: 2px solid;
       background-color: color('white');
```

Comment:
Moet deze altijd wit zijn of kan het via de themify functie lopen?

Reply:
Moet altijd wit zijn, is zelfde als page background.